pkg/grablogs/grab_logs.go

```diff
@@ -0,0 +1,65 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+package grablogs
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/redhat-partner-solutions/vse-sync-testsuite/pkg/utils"
+)
+
+var (
+	ptpContainers = []string{"gpsd", "linuxptp-daemon-container"}
```

Comment:
Please can you remove `gpsd` from this list, we don't actually need it and it's not part of the official operator.